suricata.yaml.in

```diff
@@ -621,6 +621,7 @@ af-packet:
     #  more info.
     # Recommended modes are cluster_flow on most boxes and cluster_cpu or cluster_qm on system
     # with capture card using RSS (requires cpu affinity tuning and system IRQ tuning)
+    # cluster_rollover has been deprecated; if used, it'll be replaced with cluster_flow.
```

Comment:
I read this as "in the future, it will be replaced by ...". Maybe reword a bit... "this now acts as cluster_flow" or something like this?